### ISSUE TITLE
Search (Ctrl-F): Remove "wrap around" checkbox, explicitly ask for start-over

### ISSUE THREAD

**Issue by dvtomas** [guillep/pharo-core#5950]
IMO the experience is much nicer if the search dialog asks you "No more occurences of xxx have been found, start searching again from the beginning?"

 - No need for checking the checkbox
 - Explicit feedback on starting over
 - Principle of least surprise, this is how the majority of editors do it today (Citation needed :-) )

**Comment by Ducasse** [comment]
I do not think that we will have the time to work on it. First because the search is working and second because we have many issues that require our attention.
